extensions/stats/plugin.cc

```diff
@@ -47,54 +47,63 @@ constexpr long long kDefaultTCPReportDurationMilliseconds = 15000;  // 15s
 
 namespace {
 
+// Efficient way to assign flatbuffer to a vector of strings
+#define FB_ASSIGN(name, v) \
```

Comment:
This will make istio_dimensions_ take over ownership of a subpart of flat node? So this assumes each dimension will only be set once in each onLog call, right?

Reply:
It will copy into the dimension vector. The vector owns all its content. We're just saving on local allocation (which C++17 gives for free awkward for now). It would be interesting to avoid even that copy but that requires extra care in ensuring the longevity of the flatbuffers.